From 850756980f7fb61ac872295ee1aeb5025a9243af Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Mon, 24 Feb 2025 00:40:31 -0600
Subject: [PATCH] build: Add PEP 517/518 build-system metadata

* Add pyproject.toml with build-system table for setuptools as the build backend.
   - c.f. https://learn.scientific-python.org/development/guides/packaging-classic/
---
 pyproject.toml | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 pyproject.toml

diff --git a/pyproject.toml b/pyproject.toml
new file mode 100644
index 000000000..638dd9c54
--- /dev/null
+++ b/pyproject.toml
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
-- 
2.48.1

